sklearn/cluster/_optics.py

```diff
@@ -81,7 +81,7 @@ class OPTICS(ClusterMixin, BaseEstimator):
           'manhattan']
 
         - from scipy.spatial.distance: ['braycurtis', 'canberra', 'chebyshev',
-          'correlation', 'dice', 'hamming', 'jaccard', 'kulsinski',
+          'correlation', 'dice', 'hamming', 'jaccard', 'kulsinski', 'kulczynski1',
```

Comment:
Should we also deprecate `'kulsinski'` in scikit-learn as well?

Reply:
Certainly: https://github.com/scipy/scipy/issues/2009
I even think that scikit-learn could be the reason for having this metric in SciPy.

For this PR, I would like just to add support for this metric and deal with the SciPy part. Then, we can see how the deprecation would go for the scikit-learn part and see if we need to backport something for sometimes.